### PR TITLE
Contributions from round 7

### PR DESCRIPTION
**Pull request by eileen-kuehn**
This PR adds the contributions from abstract round 7 as well as contribution with id 35.

### CODE REVIEW

**Comment by Chilipp** [comment]
thanks @eileen-kuehn! the abstract of `_events/talks/event-039.md` is a bit hard to read. Should we stick to it or ask the talks convener to change it?

File: _events/talks/event-039.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+language: English
+prerequisites: "It could be of help if attendees have already participated  in the development of research software."
+date: 2021-01-13
+---
+The goal of the proposed talk is to present the publication
+Gomez-Diaz T. and Recio T.,
+On the evaluation of research software: the CDUR procedure
+[version 2 ; peer review: 2 approved] 26 Nov 2019.
+F1000Research 2019, 8:1353,
+https://f1000research.com/articles/8-1353
```
Comment (Chilipp) [inline _events/talks/event-039.md:26]: this should be a link

```suggestion
[https://f1000research.com/articles/8-1353](https://f1000research.com/articles/8-1353)
```

**Comment by Chilipp** [comment]
@TeriForey, is it with purpose that your poster does not show your affiliation (_Teri Apps_) that you used for your last talk?

**Comment by TeriForey** [comment]
Excellent spot @Chilipp! I'm not sure why it got missed off this time but I'm happy to update it. Also, should it be in the poster section since my submission is a lightning talk + blog post?

**Comment by Chilipp** [comment]
thanks @TeriForey. yes, please. feel free to update it yourself (here's the link to your previous contribution, if you want to copy-paste: https://github.com/SORSE/sorse.github.io/blob/master/_events/talks/event-014.md)

> Also, should it be in the poster section since my submission is a lightning talk + blog post?

Yes, posters stands for _Posters and Lightning talks _, see https://sorse.github.io/programme/posters/

**Comment by eileen-kuehn** [comment]
> thanks @eileen-kuehn! the abstract of `_events/talks/event-039.md` is a bit hard to read. Should we stick to it or ask the talks convener to change it?

Yes, this might be a good idea. Do you know who was the convener this time?

**Comment by Chilipp** [comment]
> Yes, this might be a good idea. Do you know who was the convener this time?

I already asked on slack @eileen-kuehn and made some changes (https://github.com/SORSE/sorse.github.io/pull/421/commits/eab0a441422656a232f693de0ec9af1acd05daf0)

**Comment by Chilipp** [comment]
@TeriForey when will you have time to update the affiliation? Or can I just use your old one?

**Comment by TeriForey** [comment]
Sorry @Chilipp it's been a mad day - I should have time this evening, but if you want it done quicker than that then feel free to use my old affiliation

**Comment by Chilipp** [comment]
> Sorry @Chilipp it's been a mad day - I should have time this evening, but if you want it done quicker than that then feel free to use my old affiliation

sure, no problem

**Comment by Chilipp** [comment]
done with 1380a11 @TeriForey

**Comment by TeriForey** [comment]
Thanks!

**Review by Chilipp — APPROVED** [review]

**Comment by Chilipp** [comment]
alright, DOIs assigned and PDFs generated, so let's merge this

**Pull request merged**